libs/plugins/stream-server/src/lib/is-stream-resource.ts

```diff
@@ -0,0 +1,6 @@
+import { Resource } from '@flogo-web/core';
+import { RESOURCE_TYPE } from './constants';
+
+export function isStreamResource(r: Partial<Resource>): boolean {
```

Comment:
We can use the type instead of a regular boolean, this way it will also help with typescript casting:

```suggestion
export function isStreamResource(r: Partial<Resource>): r is Resource<StreamData> {
```